include/aikido/planner/World.hpp

```diff
@@ -39,6 +39,11 @@ class World
   /// \param name Name of desired Skeleton
   dart::dynamics::SkeletonPtr getSkeleton(const std::string& name) const;
 
+  /// Find a Skeleton by Skeleton, returning null if it is not in the World
+  /// \param skel Desired Skeleton
+  dart::dynamics::SkeletonPtr getSkeleton(
```

Comment:
This function sounds like checking whether the skeleton exists in (registered to) this world. I would like to suggest changing the function name `containsSkeleton(...)` or `existsSkeleton(...)` (or `skeletonExists`) (we conventionally suffix `s` to the leading verb for an inquiry function) like [`NewtonsMethodProjectable::contains`](https://github.com/personalrobotics/aikido/blob/cbf1e5808325f75b3831a62760340e3a6d7261b3/include/aikido/constraint/NewtonsMethodProjectable.hpp#L45) and [`CatkinResourceRetriever::exists(...)`](https://github.com/personalrobotics/aikido/blob/2c24a9bd10a8412084d8979a4995a81fd4103259/include/aikido/io/CatkinResourceRetriever.hpp#L33) and returning `bool`.

I personally prefer `contains` over `exists` because `contains` would sound more natural when we use the function like:
```cpp
if (world.containsSkeleton(someSkeleton))
  ...
if (world.existsSkeleton(someSkeleton)) // or if(world.skeletonExists(...))
  ...
```